ballerina-tests/tests/http2_mutual_ssl_test.bal

```diff
@@ -97,8 +97,63 @@ http:ClientConfiguration http2MutualSslClientConf = {
 };
 
 @test:Config {}
-public function testHttp2MutualSsl() {
-    http:Client httpClient = checkpanic new("https://localhost:9204", http2MutualSslClientConf);
+public function testHttp2MutualSsl1() {
+    http:Client httpClient = checkpanic new("https://localhost:9204", http2MutualSslClientConf1);
+    http:Response|error resp = httpClient->get("/http2Service/");
+    if (resp is http:Response) {
+        assertTextPayload(resp.getTextPayload(), "Passed");
+    } else {
+        test:assertFail(msg = "Found unexpected output: " +  resp.message());
+    }
+}
+
+http:ClientConfiguration http2MutualSslClientConf2 = {
+    secureSocket: {
+        key:{
+            path: "tests/certsandkeys/ballerinaKeystore.p12",
+            password: "ballerina"
+        },
+        protocol:{
+            name: http:TLS,
+            versions: ["TLSv1.2", "TLSv1.1"]
+        },
+        ciphers: ["TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA"]
+    },
+    httpVersion: "2.0",
+    http2Settings: { http2PriorKnowledge: true }
+};
+
+@test:Config {}
+public function testHttp2MutualSsl2() {
+    http:Client httpClient = checkpanic new("https://localhost:9204", http2MutualSslClientConf2);
+    http:Response|error resp = httpClient->get("/http2Service/");
+    if (resp is http:Response) {
```

Comment:
better to follow ballerina anti patterns 

https://docs.google.com/document/d/1y6QVqaZzZt9jMpYV4jP5WRS_W_KoC4y40Uuoh1ALu8E/edit

like in here, we can omit parenthesis around the if condition. `if resp is http:Response`

Reply:
That need to be done across the module itself. And also across the whole standard library. Added the missing test cases with matched to the test cases we already have.

Reply:
The issue is created to tract this. https://github.com/ballerina-platform/ballerina-standard-library/issues/1718